### ISSUE TITLE
Improve Dualshock 4 support - rumble and default button map

### ISSUE THREAD

**Issue by daniel-toman** [autobleem/AutoBleem#347]
**TL;DR — Project Eris kernel supports rumble in Dualshock 4v2 (CUH-ZCT2) controller, but not Autobleem kernel. Request adding rumble support to Autobleem kernel for this controller as well. Also suggest adding native EvoUI support for this common controller to avoid the button mapping screen.**

Hello, I have a current Dualshock 4 controller (CUH-ZCT2) aka Dualshock4v2, this is the newer revision released late 2016. I bought it around the time I bought my PS Classic, it's the only Sony Dualshock controller currently for sale (other than the new Dualsense 5).

I am unable to get rumble / vibration working in RetroArch using this controller on Autobleem 0.9.0, plugged in with USB to front port 1. Neither the stock kernel nor the Autobleem kernel will cause this controller to vibrate in Metal Gear Solid "Vibration Test" for example. I have enabled all relevant settings I could find in RetroArch:

> Pad 1: dualshock
> Pad 1 bind: RetroPad w/Analog
> Haptic Feedback/Vibration: all on

With the same RetroArch settings, I am able to plug in the 8bitdo SN30 Pro+ controller, set to MacOS mode (start+A = Dualshock 4 emulation), and "Vibration Test" in Metal Gear Solid will vibrate the 8bitdo controller. So the RetroArch settings seem correct for rumble, but the same settings don't work on the Dualshock.

This is the same behavior reported elsewhere, for example in [this controller compatibility chart.](https://www.reddit.com/r/PlaystationClassic/comments/amq0uy/updated_controller_compatibility_status_for/) Supposedly, the Dualshock 4v1 (CUH-ZCT1) 2013-2016 controller will rumble with RetroArch, but not the newer CUH-ZCT2.

HOWEVER! When I installed the Project Eris 1.0 kernel, I found that my same Dualshock 4v2 controller DOES RUMBLE in Metal Gear Solid "Vibration Test" in Project Eris' RetroArch. Even booting Autobleem 0.9.0 with the Project Eris kernel installed will rumble the Dualshock 4v2 in Autobleem's RetroArch!

So it appears there is something in the Project Eris kernel that enables rumble on the Dualshock 4v2 controller, and I'd love to see this rumble support added to the Autobleem kernel. Analog and rumble were such a big part of my Playstation experience back in the day, the PS Classic just doesn't feel right without both.

I also found that Project Eris 1.0 recognizes the Dualshock 4v2 controller out of the box, no button mapping screen appears. This was a nicer experience than Autobleem 0.9 which required me to map all the buttons after first boot. For some reason on my first attempt it recognized each button press as a double press, so half the buttons got mapped incorrectly. It was a minor inconvenience to find the configuration file on the USB drive and delete it, but would be nice if Autobleem recognized this controller out of the box. Since the Dualshock 4v2 is very common, it would make Autobleem an even nicer experience if the button mapping and rumble worked automatically. Most casual players like me would likely assume the current model Dualshock 4 controller is the best choice to add analog controls to their PS Classic.

Thanks for all your work and for considering the request!
